my-profile-webapp/src/main/webapp/my-app.js

```diff
@@ -185,6 +245,24 @@
               });
           }
           
+          var searchUsersLastName = function(searchTerm){          
+            return $http.get('/portal/api/people.json?searchTerms%5B%5D=sn&sn=' + searchTerm).success(
```

Comment:
Did you try this for anyone who is not an admin? I think there is funky security around this.